integration/metrics_collection_test.go

```diff
@@ -0,0 +1,180 @@
+package integration_test
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/sethvargo/go-retry"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	addonsv1alpha1 "github.com/openshift/addon-operator/apis/addons/v1alpha1"
+	"github.com/openshift/addon-operator/integration"
+	"github.com/openshift/addon-operator/internal/controllers"
```

Comment:
@pepedocs with the file structure changes in your branch , this needs to be changed to "github.com/openshift/addon-operator/controllers"